pkg/plugins/codegen/tmpl/autogen_header.tmpl

```diff
@@ -0,0 +1,28 @@
+{{ if .GenLicense -}}
```

Comment:
should be able to get rid of this if you're using the slash header injector approach